bootstrap/web/iframe-adapter.js

```diff
@@ -14,24 +14,29 @@
    With ZLUX, there's a global called ZoweZLUX which holds useful tools. So, a site
    Can determine what actions to take by knowing if it is or isnt embedded in ZLUX via IFrame.
 */
-
 var responses = {};
 var these = {}; //contains this'
 var contextMenuActions = {};
 var closeHandlers = {};
+var pluginDef = undefined;
+var launchMetadata = undefined;
 var curResponseKey = 0;
 var numUnresolved = 0;
 var instanceId = -1;
 
 let messageHandler = function(message) {
-    if(message.data.dispatchData){
-        if(instanceId === -1 && message.data.dispatchData.instanceId !== undefined){
-            instanceId = message.data.dispatchData.instanceId;
+    let data = message.data;
+    if(data.dispatchData){
+        if(instanceId === -1 && data.dispatchData.instanceId !== undefined){
```

Comment:
This should have an else{} which logs a warning, because it would be very weird if the desktop tried to tell you that your instance ID was changing.